### ISSUE TITLE
Avaliar alternativas melhores de como conciliar os dois sistemas

### ISSUE THREAD

**Issue by laurogripa** [Makers-Network/agenda-saude#80]
Esse repositório que hoje chamamos de "Agenda Saúde" começou como um sistema de agendamento para vacinas, chamado "Vacina Joinville" na época.

Quando surgiu a demanda de criar um sistema de agendamento para testes rápidos de COVID, decidimos criar outra branch chamada `master-covid`, para atender a demanda o mais rápido possível.

Basicamente, criamos dois sistemas, um sistema de agendamento de vacinas que é hospedado a partir da `master` e outro sistema de agendamento de testes rápidos hospedado a partir da `master-covid`

Os dois sistemas tem propósitos diferentes e grandes diferenças em partes do código, porém, ainda sim compartilham boa parte do código.

Precisamos pensar em uma maneira melhor de lidar com a existência dos dois sistemas, pois quanto mais mudanças tem em cada sistema, mais difícil fica gerenciar o código compartilhado.

No **longo prazo** uma possibilidade é extrair o código compartilhado e criar uma solução "white-label" customizável.
No **curto prazo** uma possibilidade é separar em dois repositórios, mesmo que compartilhem bastante código.

O que acham?

**Comment by laurogripa** [comment]
Sugestão do @pedroCervi : 

```
Sugestão em detalhes (2 repos): podemos fazer um fork do repo original (vacina-joinville), pegar todos os commits do app de covid e passar a iterar com dois repos, sendo que um é fork. Ai oq for geral comitamos no vacina-joinville e depois vamos atualizando o fork. Podemos fazer isso junto com a estratégia de migração para a Digital Ocean daí.
```

**Comment by pedroCervi** [comment]
teste

**Comment by edrd-f** [comment]
Design doc: https://hackmd.io/ntPqhK31Qw64OvwFQN4YfA

**Comment by jmonteiro** [comment]
Concordo, porém esperaria até o sistema estar mais completo. Por hora ainda estamos adicionando novas funcionalidades com frequencia. Passando o "test do COVID", este app será uma ótima ferramenta para vacinação em geral.
